diff --git a/Stored Procedures/dbo.spGetOptionItem.sql b/Stored Procedures/dbo.spGetOptionItem.sql
index c6f65ae..6a0f64d 100644
--- a/Stored Procedures/dbo.spGetOptionItem.sql
+++ b/Stored Procedures/dbo.spGetOptionItem.sql
@@ -2,28 +2,32 @@
 GO
 SET ANSI_NULLS ON
 GO
-
 -- =============================================
 -- Author:		<Dorothy Baum>
 -- Create date: <June 19,2009>
 -- Description:	<Return the OptionValue for OptionItem, ProgramCode, for specify date>
+-- mod: <Jay Robohn> <Aug 06,2018> <Make ProgramFK optional>
 -- =============================================
-create PROCEDURE [dbo].[spGetOptionItem]
-	(@OptionItem varchar(50),@ProgramFK int,@CompareDate datetime, @OptionValue varchar(200) output)
-	
-AS
-BEGIN
-	-- SET NOCOUNT ON added to prevent extra result sets from
-	-- interfering with SELECT statements.
-	SET NOCOUNT ON;
+CREATE procedure [dbo].[spGetOptionItem] (@OptionItem varchar(50)
+								, @ProgramFK int
+								, @CompareDate datetime
+								, @OptionValue varchar(200) output)
+as
+	begin
+		-- SET NOCOUNT ON added to prevent extra result sets from
+		-- interfering with SELECT statements.
+		set noCount on ;
 
-    -- Insert statements for procedure here
-	SELECT @OptionValue = OptionValue FROM appOptions 
-    WHERE OptionItem  = @OptionItem and 
-		  ProgramFK = @ProgramFK and 
-		  @CompareDate between 
-		  OptionStart and isnull(OptionEnd,getdate())
+		-- Insert statements for procedure here
+		select	@OptionValue = OptionValue
+		from	AppOptions
+		where	OptionItem = @OptionItem 
+				and case when @ProgramFK is null then 1
+						when ProgramFK = @ProgramFK then 1
+						else 0
+					end = 1
+				and @CompareDate between OptionStart and isnull(OptionEnd, getdate())
 
-END
+	end ;
 
 GO
diff --git a/Tables/dbo.ASQSE.sql b/Tables/dbo.ASQSE.sql
index 37e8235..939ab71 100644
--- a/Tables/dbo.ASQSE.sql
+++ b/Tables/dbo.ASQSE.sql
@@ -11,7 +11,7 @@
 [ASQSEReceiving] [char] (1) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
 [ASQSEReferred] [char] (1) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
 [ASQSETCAge] [char] (2) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
-[ASQSETotalScore] [numeric] (4, 1) NOT NULL,
+[ASQSETotalScore] [numeric] (3, 0) NOT NULL,
 [ASQSEVersion] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
 [DiscussedWithPC1] [bit] NULL,
 [FSWFK] [int] NOT NULL,
diff --git a/Tables/dbo.ASQSEDeleted.sql b/Tables/dbo.ASQSEDeleted.sql
new file mode 100644
index 0000000..a4ae24e
--- /dev/null
+++ b/Tables/dbo.ASQSEDeleted.sql
@@ -0,0 +1,28 @@
+CREATE TABLE [dbo].[ASQSEDeleted]
+(
+[ASQSEDeletedPK] [int] NOT NULL IDENTITY(1, 1),
+[ASQSEPK] [int] NOT NULL,
+[ASQSECreateDate] [datetime] NOT NULL,
+[ASQSECreator] [char] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[ASQSEDateCompleted] [datetime] NOT NULL,
+[ASQSEDeleteDate] [datetime] NULL,
+[ASQSEDeleter] [char] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ASQSEEditDate] [datetime] NULL,
+[ASQSEEditor] [char] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ASQSEInWindow] [bit] NOT NULL,
+[ASQSEOverCutOff] [bit] NOT NULL,
+[ASQSEReceiving] [char] (1) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ASQSEReferred] [char] (1) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ASQSETCAge] [char] (2) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[ASQSETotalScore] [numeric] (4, 1) NOT NULL,
+[ASQSEVersion] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[DiscussedWithPC1] [bit] NULL,
+[FSWFK] [int] NOT NULL,
+[ReviewCDS] [bit] NOT NULL,
+[HVCaseFK] [int] NOT NULL,
+[ProgramFK] [int] NOT NULL,
+[TCIDFK] [int] NOT NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[ASQSEDeleted] ADD CONSTRAINT [PK__ASQSEDeleted] PRIMARY KEY CLUSTERED  ([ASQSEDeletedPK]) ON [PRIMARY]
+GO
diff --git a/Tables/dbo.CommonAttributes.sql b/Tables/dbo.CommonAttributes.sql
index e363824..aa821d4 100644
--- a/Tables/dbo.CommonAttributes.sql
+++ b/Tables/dbo.CommonAttributes.sql
@@ -38,7 +38,7 @@
 [OBPInHome] [char] (1) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
 [OBPInvolvement] [char] (2) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
 [OBPInvolvementSpecify] [varchar] (500) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
-[Parity] [char] (2) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[Parity] [int] NULL,
 [PBEmergencyAssistance] [char] (1) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
 [PBEmergencyAssistanceAmount] [numeric] (4, 0) NULL,
 [PBFoodStamps] [char] (1) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
@@ -99,6 +99,8 @@
 GO
 ALTER TABLE [dbo].[CommonAttributes] ADD CONSTRAINT [PK__CommonAt__14761E7359FA5E80] PRIMARY KEY CLUSTERED  ([CommonAttributesPK]) ON [PRIMARY]
 GO
+CREATE NONCLUSTERED INDEX [IX_CommonAttributes_FormType] ON [dbo].[CommonAttributes] ([FormType]) INCLUDE ([EducationalEnrollment], [FormFK], [HighestGrade], [IsCurrentlyEmployed], [PrimaryLanguage]) ON [PRIMARY]
+GO
 CREATE NONCLUSTERED INDEX [IX_FK_CommonAttributes_HVCaseFK] ON [dbo].[CommonAttributes] ([HVCaseFK]) ON [PRIMARY]
 GO
 CREATE NONCLUSTERED INDEX [IX_FK_CommonAttributes_PC1MedicalFacilityFK] ON [dbo].[CommonAttributes] ([PC1MedicalFacilityFK]) ON [PRIMARY]
diff --git a/Tables/dbo.scoreASQSE.sql b/Tables/dbo.scoreASQSE.sql
index 4c20375..465b7dc 100644
--- a/Tables/dbo.scoreASQSE.sql
+++ b/Tables/dbo.scoreASQSE.sql
@@ -3,7 +3,7 @@
 [scoreASQSEPK] [int] NOT NULL IDENTITY(1, 1),
 [ASQSEVersion] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
 [MaximumASQSEScore] [numeric] (3, 0) NOT NULL,
-[SocialEmotionalScore] [numeric] (6, 2) NOT NULL,
+[SocialEmotionalScore] [numeric] (3, 0) NOT NULL,
 [TCAge] [char] (4) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL
 ) ON [PRIMARY]
 GO
